ui/qml/pages/UnpairDeviceDialog.qml

```diff
@@ -0,0 +1,46 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import uk.co.piggz.amazfish 1.0
+
+Dialog {
+    acceptDestination: Qt.resolvedUrl("PairSelectDeviceType.qml")
+    acceptDestinationAction: PageStackAction.Replace
+
+    onAccepted: {
+        DaemonInterfaceInstance.disconnect()
+        AmazfishConfig.pairedAddress = ""
+        AmazfishConfig.pairedName = ""
+        AmazfishConfig.deviceAuthKey = ""
```

Comment:
Im not sure about wiping the auth key....if a user decides to re-pair (like me while testing!) they will have to enter the whole key again which is a pain.  thoughts?

Reply:
Possible solution is in the last commit